programs/svm-spoke/src/instructions/slow_fill.rs

```diff
@@ -252,3 +311,26 @@ pub fn execute_v3_slow_relay_leaf<'info>(
 
     Ok(())
 }
+
+// Helper to unwrap optional instruction params with fallback loading from buffer account.
+fn unwrap_execute_v3_slow_relay_leaf_params(
+    slow_fill_leaf: Option<V3SlowFill>,
+    proof: Option<Vec<[u8; 32]>>,
+    account: &Option<Account<ExecuteV3SlowRelayLeafParams>>,
+) -> ExecuteV3SlowRelayLeafParams {
+    match (slow_fill_leaf, proof) {
+        (Some(slow_fill_leaf), Some(proof)) => ExecuteV3SlowRelayLeafParams {
+            slow_fill_leaf,
+            root_bundle_id: 0, // This is not used in the caller.
+            proof,
+        },
+        _ => account
+            .as_ref()
+            .map(|account| ExecuteV3SlowRelayLeafParams {
+                slow_fill_leaf: account.slow_fill_leaf.clone(),
+                root_bundle_id: account.root_bundle_id,
```

Comment:
Is this `root_bundle_id` used by the caller? do we need it? Why in the fallback we return something and if in the other case we return 0?